### PR TITLE
keytool: display anemo::PeerId for ed25519 keys

### PR DESCRIPTION
**Pull request by bmwill**

### CODE REVIEW

File: crates/sui/src/keytool.rs [hunk 1]
```diff
@@ -135,6 +135,10 @@ impl KeyToolCommand {
                     Ok(keypair) => {
                         println!("Public Key: {}", keypair.public().encode_base64());
                         println!("Flag: {}", keypair.public().flag());
+                        if let PublicKey::Ed25519(public_key) = keypair.public() {
```
Comment (kchalkias) [inline crates/sui/src/keytool.rs:138]: Do we enforce validators using different anemo keys? or in theory they can "on purpose" share one?

Comment (bmwill) [reply crates/sui/src/keytool.rs:138]: we only support ed25519 for anemo today

Comment (bmwill) [reply crates/sui/src/keytool.rs:138]: This is also true of any of our keys that i'm aware of

**Review by kchalkias — APPROVED** [review]
Approving this for now and we discuss the duplicated key in another thread.

File: crates/sui/src/keytool.rs (hunk 1 above)
Comment (kchalkias) [reply crates/sui/src/keytool.rs:138]: opened an issue here: https://github.com/MystenLabs/sui/issues/9351

**Pull request merged**